engine/JustDecompiler.Shared/Languages/AttributeWriter.cs

```diff
@@ -576,7 +576,7 @@ private void WriteGlobalAttribute(CustomAttribute attribute, string keyword)
             {
                 genericWriter.Write("    ");
                 string comment = genericWriter.Language.CommentLines(ASSEMBLYNOTRESOLVEDERROR);
-                genericWriter.Write(comment.Remove(comment.Length - 2));
+                genericWriter.Write(comment.Remove(comment.Length - Environment.NewLine.Length));
```

Comment:
Changes to existing code should be wrapped with license comments:

```suggestion
                /* AGPL */
                genericWriter.Write(comment.Remove(comment.Length - Environment.NewLine.Length));
                /* End AGPL */
```